static/flow/views/program-editor-panel.js

```diff
@@ -84,7 +84,7 @@ var ProgramEditorPanel = function(options) {
             programSpec.displayedName = "untitled program";
             programSpec.archived = false;
         }
-        else if(programSpec.name = ""){
+        else if(programSpec.name == ""){
```

Comment:
I'd probably squash commits like this and the previous one in the future, just so it's less likely anyone accidentally reverts to a broken version.